src/feltor/feltor.cu

```diff
@@ -194,17 +194,20 @@ int main( int argc, char* argv[])
     }
     /// /////////////The initial field//////////////////////////////////////////
     double time = 0.;
+    std::vector<double> time_intern(p.itstp);
     Vector y0;
     std::array<dg::x::DVec, 3> gradPsip;
+    dg::geo::Nablas<dg::x::CylindricalGrid3d> nabla(grid);
     gradPsip[0] =  dg::evaluate( mag.psipR(), grid);
     gradPsip[1] =  dg::evaluate( mag.psipZ(), grid);
     gradPsip[2] =  dg::evaluate( dg::zero, grid); //zero
-    unsigned failed = 0;
     feltor::Variables var{
-        feltor, y0, p, mag, gradPsip, gradPsip,
+        feltor, y0, p, mag, nabla, gradPsip, gradPsip, gradPsip, gradPsip,
         dg::construct<dg::x::DVec>( dg::pullback( dg::geo::Hoo(mag),grid)),
         0., // duration
-        &failed // nfailed
```

Comment:
do not delete this line, Variables expects a pointer to double as the last argument